content/blog/2020-04-27-moving-to-the-matrix/index.md

```diff
@@ -0,0 +1,163 @@
+
+Some other criteria fell under the nice-to-have category:
+
+- All things equal, we would certainly prefer an open source solution to a
+  proprietary one.
+- An open standard for interoperability was considered important, if not
+  essential. This would allow easy interaction, e.g., between IRC and our chat
+  service.
+- Desktop clients and terminal-based clients were considered a great plus.
+- Ideally, this is a service that is used by other organizations as well.
+- GitHub integration and bells and whistles similar to that were considered a
+  plus as well.
+
+Using these criteria, we surveyed a bunch of options, including Mattermost,
+sticking with Slack, Discord, gitter, and of course Matrix.
+
+To test not only chat solutions, but also hosters, we rented a Mattermost and
+Matrix instance from a [Swiss hoster](https://unglei.ch) and tested them within
+a closed circle. Other services we could test with a free plan from their
+respective providers.
+
+After several weeks of trial, we conceded that Matrix ticked most of the boxes:
+
+- It is fully open-source, and committed to open standards
+- Matrix is used by many organizations such as KDE, Gnome, the Wikimedia Foundation,
+  and the French government, to name a few.
+- Matrix instances can connect to each other ("federation"). This means from our
+  GNU Radio chat, we can access other organizations on Matrix, even if they're
+  not part of the GNU Radio chat instance. This includes the main Matrix instance
+  on [matrix.org](https://matrix.org), where projects such as our friends from
+  SatNOGS are located.
+- That means that people already on Matrix don't have to do a whole lot to join
+  our Matrix server.
+- Bridging to IRC is also easy, and will have a high likelihood of working
+  indefinitely.
+- The open standards nature of Matrix, and the availability of open source
+  reference clients, means that there are lots of clients out there, including
+  things like a [Weechat plugin](https://github.com/poljar/weechat-matrix).
+
+
+## The matter of Transparency, or why am I only hearing about this now?
+
+While the GNU Radio leadership team definitely did their due diligence in
+picking a new service, one thing that definitely shouldn't become the norm for
+decisions around the GNU Radio project is the fact that we decided this behind
+closed doors within the leadership team. For example, we did not publish a GREP
+ahead of time for a public discussion.
+
+In this case, we were under a bit of time pressure, and the choice of chat service
+seems to be something where a lot of personal preference comes in. To be blunt,
+if we were going to allow everyone to argue on behalf of their favourite messenger,
+we wouldn't have been able to resolve this on time.
+
+## What happens to the GNU Radio Slack?
+
+The Slack plan we're on right now is free, and there's no reason to turn it off.
+However, all important discussions regarding the project will be held on our
```

Comment:
However, the main development, documentation, and variety of other discussions regarding the project...